hiyapyco/__init__.py

```diff
@@ -366,11 +396,21 @@ def _substmerge(self, a, b):
             a = copy.deepcopy(a)
             b = copy.deepcopy(b)
         logger.debug('>' * 30)
-        logger.debug('substmerge %s and %s' % (a, b,))
-        # FIXME: make None usage configurable
+        logger.debug(
```

Comment:
I do not think that this change is related to the topic of the PR and it does not improve readability ... pls revert this change

Reply:
sorry, this was caused by the ruff autoformatter which I forgot to turn off. I like autoformatting a lot, especially because it prevents discussions like this during PR review, but I agree this change doesn't make sense in isolation.